src/plugins/proxy/src/remote_tensor.cpp

```diff
@@ -59,12 +63,23 @@ const ov::Strides& ov::proxy::RemoteTensor::get_strides() const {
     return m_tensor->get_strides();
 }
 
-const ov::SoPtr<ov::ITensor>& ov::proxy::RemoteTensor::get_hardware_tensor(const ov::SoPtr<ov::ITensor>& tensor) {
+ov::SoPtr<ov::ITensor> ov::proxy::RemoteTensor::get_hardware_tensor(const ov::SoPtr<ov::ITensor>& tensor, bool unwrap) {
```

Comment:
Looks like it still should be able to work with `const ov::SoPtr<ov::ITensor>&`... Am I wrong?

Reply:
It complains on returning reference to local